demos_community.yaml

```
@@ -1,3 +1,27 @@
+- title: Hybrid MNIST Classification Model

```

@CatalinaAlbornoz the code we're using to generate permanent links for the cards is `lastnameYEARfirstwordoftitle`. As a result, it causes both cards to have the same link, `choe2021hybrid`. Perhaps we can remove 'hybrid' from the title of this one, to avoid that happening?
```suggestion
- title: Quantum-Classical MNIST Classification Model
```